docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -105,6 +105,7 @@ If there are profile-specific YAML (or properties) files, these are also applied
 Higher precedence translates to a `PropertySource` listed earlier in the `Environment`.
 (These same rules apply in a standalone Spring Boot application.)
 
+You can set spring.cloud.config.server.acceptEmtpy to false so that Server would return a HTTP 404 status, if the application is not found .By default, this flag is set to true.
```

Comment:
@spencergibb should this be `spring.cloud.config.client.acceptEmpty` since you are configuring the client behavior?

Reply:
I don't think so, this is all server-side behavior. It should be `accept-empty`, BTW.

Reply:
Have incorporated the review comment. Also, corrected a typo in the variable name